packages/cli/src/utils/check-version.ts

```diff
@@ -3,16 +3,23 @@ import {createRequire} from 'module';
 import {checkForNewVersion} from '@shopify/cli-kit/node/node-package-manager';
 import {renderInfo} from '@shopify/cli-kit/node/ui';
 
-const PACKAGE_NAME = '@shopify/hydrogen';
+const PACKAGE_NAMES = {
+  main: '@shopify/hydrogen',
+  cli: '@shopify/cli-hydrogen',
+} as const;
 
-export async function checkHydrogenVersion(directory: string) {
+export async function checkHydrogenVersion(
+  directory: string,
+  pkgKey: keyof typeof PACKAGE_NAMES = 'main',
+) {
   if (process.env.LOCAL_DEV) return;
+  const pkgName = PACKAGE_NAMES[pkgKey];
 
   const require = createRequire(import.meta.url);
   const pkgJsonPath = locateDependency(
```

Comment:
Just double checking, this function resolves to the package in their local node_modules rather what version is listed in the project package.json?

Reply:
Yes, it resolves to the installed version, not the listed one. That's the correct behavior, right?

Reply:
Indeed! Thanks for clarifying, this looks good to me 👍